lstchain/datachecks/dl1_checker.py

```diff
@@ -1193,3 +1262,190 @@ def merge_dl1datacheck_files(file_list):
+        c.glyphs.data_source.add(cluster_j, 'cluster_j')
+        c.glyphs.data_source.add(pix_id_in_cluster, 'pix_id_in_cluster')
+
+        c.add_colorbar()
+        c.fig.plot_width = pad_width
+        c.fig.plot_height = int(pad_height * 0.85)
+        c.fig.title = Title(text=label)
+        c.fig.grid.visible = False
+        c.fig.axis.visible = True
+        c.fig.xaxis.axis_label = 'X position (m)'
+        c.fig.yaxis.axis_label = 'Y position (m)'
+        c.fig.add_tools(
+            HoverTool(tooltips=[('pix_id', '@pix_id'),
+                                ('value', '@value'),
+                                ('cluster (i,j)', '(@cluster_i, @cluster_j)'),
+                                ('pix in cluster', '@pix_id_in_cluster')],
+                      mode='mouse', point_policy='snap_to_data'))
+
+    tab1 = Panel(child=cam.fig, title='linear')
+    tab2 = Panel(child=camlog.fig, title='logarithmic')
+    p1 = Tabs(tabs=[tab1, tab2])
+
+    p2 = figure(background_fill_color='#ffffff',
+                y_range=(-0.05*cam.image.max(), cam.image.max() * 1.1),
+                x_axis_label='Pixel id',
+                y_axis_label=label)
+    p2.min_border_top = 60
+    p2.min_border_bottom = 70
+    glyphs = p2.circle(x=cam.geom.pix_id, y=cam.image, size=2)
+    glyphs.data_source.add(list(cam.geom.pix_id), 'pix_id')
+    glyphs.data_source.add(list(cam.image), 'value')
+    p2.add_tools(
+        HoverTool(tooltips=[('(pix_id, value)', '(@pix_id, @value)')],
+                  mode='mouse', point_policy='snap_to_data'))
+
+    hist, edges = np.histogram(cam.image, bins=200)
+    p3 = figure(background_fill_color='#ffffff',
+                y_range=(0.7, hist.max() * 1.1), x_axis_label=label,
+                y_axis_label='Number of pixels', y_axis_type='log')
+    p3.quad(top=hist, bottom=0.7, left=edges[:-1], right=edges[1:])
+
+    return [p1, p2, p3]
+
+
+def get_pixel_location(pix_id):
+
+    """
+
+    Parameters
+    ----------
+    pix_id pixel id number
+
+    Returns
+    -------
+    "Hardware" parameters of the pixel:
+    [cluster_i, cluster_j, pixe_id_within_cluster]
+
+    Info from https://forge.in2p3.fr/issues/33587, stored in
+    cta-lstchain/lstchain/io/LST_pixid_to_cluster.txt
+
+    """
+    if len(pixel_hardware_info) > 0:
+        return pixel_hardware_info[pix_id]
+
+    # The first time we read in the data stored in the io directory:
+    infilename = Path(Path(os.path.dirname(__file__)).parent,'io',
+                      'LST_pixid_to_cluster.txt')
+
+    with open(infilename) as file:
+        for line in file:
+            if line.strip()[0] != '#':
+                pixel_hardware_info.append([])
+        file.seek(0)
+        for line in file:
+            if line[0] == '#':
+                continue
+            data = [int(s) for s in line.split()]
+            pixel_hardware_info[data[0]] = [data[1], data[2], data[3]]
+
+    return pixel_hardware_info[pix_id]
```

Comment:
looking at the `LST_pixid_to_cluster.txt`, wouldn't it be better to use something like:
```
data = np.genfromtxt(infilename, comments='#')
```

Reply:
Or pandas read_csv or read_fwf

Reply:
> data = np.genfromtxt(infilename, comments='#')

Thanks, very useful!

Reply:
If you do not comment the line with the column names, `pd.read_fwf` will directly give you a dataframe with the correct column names

Reply:
> If you do not comment the line with the column names, `pd.read_fwf` will directly give you a dataframe with the correct column names

Ok, thx for the info, but not needed here I'd say.